modules/imgproc/perf/opencl/perf_imgproc.cpp

```diff
@@ -166,7 +166,10 @@ OCL_PERF_TEST_P(CornerMinEigenValFixture, CornerMinEigenVal,
 
     OCL_TEST_CYCLE() cv::cornerMinEigenVal(src, dst, blockSize, apertureSize, borderType);
 
-    SANITY_CHECK(dst, 1e-6, ERROR_RELATIVE);
+    if (ocl::Device::getDefault().isIntel())
+        Near(1e-6, true);
```

Comment:
is it intentional that this is different from the number in test_imgproc.cpp?

Reply:
Yes.
Perf tests should not verify accuracy in the accurate way (or even they should not perform that at all)

Reply:
this is checking for more accuracy than the test_imgproc.cpp test, I can make the number here the same as test_imgproc? or I could remove the `if` and leave only `SANITY_CHECK(dst, 0.1, ERROR_RELATIVE);`